### PR TITLE
[MoM] Allow people to ask Rubik about the alien flora scattered around

### PR DESCRIPTION
**Pull request by Standing-Storm**
#### Summary
Mods "[MoM] Allow people to ask Rubik about the alien flora scattered around"

#### Purpose of change

Getting back into adding dialogue, wanted to let people ask Rubik about the patches of alien flora.

#### Describe the solution

Add some dialogue options. They don't show up until a bit of time has passed, to make sure that the player has definitely seen weird alien trees or moss before they can ask about them.

#### Testing

Topics appear and lead to appropriate places.

### CODE REVIEW

**Comment by Standing-Storm** [comment]
@I-am-Erk Did my best, but if you have a moment to look it over, I'd be thankful.

File: data/mods/MindOverMatter/npcs/dialogue/Rubik.json [hunk 1]
```diff
@@ -1,4 +1,82 @@
 [
+  {
+    "id": [ "TALK_EXODII_MERCHANT_Talk" ],
+    "type": "talk_topic",
```
Comment (Maleclypse) [inline data/mods/MindOverMatter/npcs/dialogue/Rubik.json:4]:
```suggestion
    "type": "talk_topic"
```

File: data/mods/MindOverMatter/npcs/dialogue/Rubik.json [hunk 2]
```diff
@@ -1,4 +1,82 @@
 [
+  {
+    "id": [ "TALK_EXODII_MERCHANT_Talk" ],
+    "type": "talk_topic"
```
Comment (Maleclypse) [inline data/mods/MindOverMatter/npcs/dialogue/Rubik.json:4]:
```suggestion
    "type": "talk_topic",
```

**Comment by I-am-Erk** [comment]
Sorry I missed the ping. I'm pretty behind on some dialogue reviews but I'll take a look and if something needs fixing I'll try to pr it

**Pull request merged**